db_scripts/db_dump

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+NAME=${1:-postgres}
+CONTAINER=$(docker ps --format "{{.Names}}" | grep "$NAME")
+if [ -z "$CONTAINER" ]
+	then
+		echo "Container not found"
+		exit 1
+fi
+FILE="wubloader-$(date +%Y-%m-%dT%H:%M:%S).sql"
```

Comment:
Rather than dumping to a temporary file then uploading the resulting file, we can skip the middleman and just pipe the result directly:
```
docker exec "$CONTAINER" pg_dump wubloader -U postgres | aws s3 cp - "s3://$BUCKET/$FILE"
```